js/abilities.js

```diff
@@ -25,6 +26,13 @@ var abilitiesManagement = {
             this.$forceUpdate()
 
         },
+        displayAbilitiesToExchange: function(param) {
+            if (this.chosenCardExchanger == param) {
+                this.chosenCardExchanger = null
+            } else {
+                this.chosenCardExchanger = param
+            }
+        },
         addAbility: function(card) {   
             card.duration = 0
             if (!this.abilitiesChosen.includes(card)) {
```

Comment:
When adding ability, we check that the number of cards in hand is below the maximum. 
We should use a different method for accepting new cards, as it should not perform this check.